packages/mermaid/src/diagrams/user-journey/journeyRenderer.ts

```diff
@@ -48,7 +48,9 @@ function drawActorLegend(diagram) {
 const conf = getConfig().journey;
 const LEFT_MARGIN = conf.leftMargin;
 export const draw = function (text, id, version, diagObj) {
-  const conf = getConfig().journey;
+  const titleColor = conf.titleColor;
+  const titleFontSize = conf.titleFontSize;
+  const titleFontFamily = conf.titleFontFamily;
 
   const securityLevel = getConfig().securityLevel;
```

Comment:
Using the conf object will cause issues, as it's only initialized during the initial load.
Please use the configObject as before, and remove the getConfig() call for securityLevel